src/components/commons/collections/user-menu/user-menu.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Menu } from '@material-ui/core'
+import LogoutButton from './logout-button'
+
+const UserMenu = ({anchorEl, loggedIn, onClose}) => {
+	const resources = []
+
+	if (loggedIn)
+		resources.push(<LogoutButton onClose={onClose} />)
```

Comment:
Curious about this, generally it's best to keep data to be rendered anchored on a state so React can control it's (re)rendering, but as there's no changes to resources during this state lifecycle I think this is fine (even a better approach)